oidc-controller/api/core/config.py

```diff
@@ -184,7 +184,8 @@ class GlobalConfig(BaseSettings):
         os.environ.get("USE_OOB_LOCAL_DID_SERVICE", False)
     )
     SET_NON_REVOKED: bool = bool(os.environ.get("SET_NON_REVOKED", True))
-
+    USE_V1_COMPATIBILITY: bool = bool(os.environ.get("USE_V1_COMPATIBILITY", False))
```

Comment:
Not sure if default should be true?

Reply:
I think `False` is fine: this is a migration step we will want to get away from whenever possible.